### ISSUE TITLE
Reference to nonexistent stat_bindot() in helpful message from geom_dotplot()

### ISSUE THREAD

**Issue by mdlama** [tidyverse/ggplot2#3863]
I see that `stat_bindot()` was removed in a previous issue (#1194).  The helpful message to pick a better binwidth still includes the reference to the function, as seen with this reprex.

``` r
library(ggplot2)

ggplot(mtcars, aes(x = mpg)) + geom_dotplot()
#> `stat_bindot()` using `bins = 30`. Pick better value with `binwidth`.
```

![](https://i.imgur.com/F1w7VG2.png)

``` r

stat_bindot()
#> Error in stat_bindot(): could not find function "stat_bindot"
```

<sup>Created on 2020-03-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

I also see that the ggproto object with class StatBindot is still alive and kicking.

``` r
library(ggplot2)

geom_dotplot()$stat
#> <ggproto object: Class StatBindot, Stat, gg>
#>     aesthetics: function
#>     compute_group: function
#>     compute_layer: function
#>     compute_panel: function
#>     default_aes: uneval
#>     extra_params: na.rm
#>     finish_layer: function
#>     non_missing_aes: weight
#>     parameters: function
#>     required_aes: x
#>     retransform: TRUE
#>     setup_data: function
#>     setup_params: function
#>     super:  <ggproto object: Class Stat, gg>
```

<sup>Created on 2020-03-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Out of curiosity, why was `stat_bindot()` removed?  I might be mistaken, but it seems to be the only geom with a default stat that doesn't have a `stat_*` function.

**Comment by thomasp85** [comment]
I cannot speak to what Hadley was thinking back in 2015... geom_dotplot is somewhat of an oddity since it isn't scaled in one of the directions.

We'll make sure to update the error messages - thanks for reporting it

**Comment by yutannihilation** [comment]
Fixed by #4166
